src/Maestro/Maestro.Web/.config/settings.json

```diff
@@ -27,10 +27,6 @@
       }
     ]
   },
-  "DataProtection": {
-    "KeyFileUri": "[vault(data-protection-key-file-uri)]",
```

Comment:
this is not used, the actual configuration that is used is in settings.Staging/Production